pyvisa/resources/serial.py

```diff
@@ -21,55 +21,62 @@ class SerialInstrument(MessageBasedResource):
 
     """
 
-    #: Baud rate of the interface.
+    #: Baud rate of the interface. The default value is 9600.
     baud_rate: Attribute[int] = attributes.AttrVI_ATTR_ASRL_BAUD()
 
-    #: Number of data bits contained in each frame (from 5 to 8).
+    #: Number of data bits contained in each frame (from 5 to 8). The default value is 8.
     data_bits: Attribute[int] = attributes.AttrVI_ATTR_ASRL_DATA_BITS()
 
-    #: Parity used with every frame transmitted and received.
+    #: Parity used with every frame transmitted and received. The default value is None (VI_ASRL_PAR_NONE).
     parity: Attribute[constants.Parity] = attributes.AttrVI_ATTR_ASRL_PARITY()
 
-    #: Number of stop bits used to indicate the end of a frame.
+    #: Number of stop bits used to indicate the end of a frame. The default value is 1 (VI_ASRL_STOP_ONE).
     stop_bits: Attribute[constants.StopBits] = attributes.AttrVI_ATTR_ASRL_STOP_BITS()
 
-    #: Indicates the type of flow control used by the transfer mechanism.
+    #: Indicates the type of flow control used by the transfer mechanism. The default value
+    #: is None (VI_ASRL_FLOW_NONE).
```

Comment:
Python None is not acceptable for those attributes. Rather than the bare constants I would prefer to mention the enumeration `constants.ControlFlow.none`

Reply:
You could also fix it in attributes.py if you get a chance